src/js/components/DateInput/__tests__/DateInput-test.js

```diff
@@ -369,7 +351,8 @@ describe('DateInput', () => {
 
   test(`buttonProps should pass props to Button 
   when not inline and no format`, () => {
-    const component = renderer.create(
+    window.scrollTo = jest.fn();
```

Comment:
Why was `window.scrollTo` included here? It is not affecting the test or snapshot

Reply:
Drop calls it via `setFocusWithoutScroll`. In the older `renderer` approach, we weren't going as far rendering, so this didn't show up. You can see how we do the same thing in Drop-test, Select-test, etc.

Reply:
Ah okay thanks for the explanation